terraform/production/main.tf

```diff
@@ -0,0 +1,74 @@
+terraform {
+  required_version = "~> 0.13.2"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "3.5.0"
+    }
+
+    circleci = {
+      source  = "mrolla/circleci"
+      version = "0.4.0"
+    }
+
+    cloudfoundry = {
+      source  = "cloudfoundry-community/cloudfoundry"
+      version = "0.12.6"
+    }
+  }
+
+  backend "s3" {
+    # The values here MUST be hardcoded
+    # AWS credentials MUST be provided by environment variables or credentials file
+    bucket         = "federalist-terraform"
+    key            = "production/terraform.tfstate"
+    region         = "us-gov-west-1"
+    dynamodb_table = "federalist-terraform-locks"
+    encrypt        = true
+  }
+}
+
+# provider "aws" {
```

Comment:
is this to be filled out during setup? there was a reference to such in the readme